packages/devtools_app/web/debugger_screen.html

```diff
@@ -39,7 +39,9 @@
 </head>
 
 <body layout vertical style="width: 100%; height: 100%; background-color: transparent;">
```

Comment:
There might be quite a flash of oddly styled content when this screen is displayed. As we can't remove the content until 1259 is landed in a separate cl, at least make all the unstyled content be hidden by default.

Reply:
Everything except for the content div is already set to hidden.

The styles will be loaded before any of the other html content of this page, so I'm not sure what would come up as oddly styled.

I can set the entire body to be hidden until the framework loads the debugger screen, but I don't notice any difference between that and making no change.

Reply:
that is all I was looking for